pkg/tests/api/alerting/api_alertmanager_test.go

```diff
@@ -0,0 +1,133 @@
+package alerting
+
+import (
+	"context"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"testing"
+	"time"
+
+	"github.com/grafana/grafana/pkg/services/ngalert/models"
+	"github.com/grafana/grafana/pkg/services/sqlstore"
+
+	"github.com/grafana/grafana/pkg/tests/testinfra"
+	"github.com/stretchr/testify/require"
+)
+
+func TestAlertAndGroupsQuery(t *testing.T) {
+	dir, path := testinfra.CreateGrafDir(t, testinfra.GrafanaOpts{
+		EnableFeatureToggles: []string{"ngalert"},
+	})
+	store := setupDB(t, dir)
+	grafanaListedAddr := testinfra.StartGrafana(t, dir, path, store)
+
+	// When there are no alerts available, it returns an empty list.
```

Comment:
We currently have no way to introduce alerts via the API (it is deliberately disabled) and I didn't want to have to go through creating an alert and waiting for it to come up. 

For now, this is enough as we'll revisit this soon with the silences tests.